recipes/impyla/meta.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: impyla
+  version: "0.13.5"
+
+source:
+  git_url: https://github.com/cloudera/impyla
+  git_rev: v0.13.6
```

Comment:
Seems like there is a version metadata issue. I see `0.13.6` above but you are fetching `0.13.6`. Take a look at the example recipe and the jinja template to define the version number. That helps avoid that error.